lib/compute/jenkins-plugins.ts

```diff
@@ -117,14 +117,11 @@ export class JenkinsPlugins {
       'rebuild',
       'resource-disposer',
       'role-strategy',
-      'ruby-runtime',
-      'rvm',
       's3',
       'saml',
       'scm-api',
       'script-security',
       'simple-theme-plugin',
-      'singleuseslave',
```

Comment:
This is being used for security review.

Reply:
Per the notice, https://www.jenkins.io/blog/2021/12/22/deprecated-ruby-runtime/.  We have switched to using disposable docker agents instead of disposable nodes from the previous iteration
```
Single Use Agent
Last released 7 years ago.
Suggestion: Use cloud agents (Fargate, Azure Container Instances, Docker, etc.) to allocate agents for a single use and then release them.
```

Reply:
Security has previous reject my opinion on removing docker node, stating removing agent is much safer option, hence I ask.

Reply:
Plugin distribution has been suspended for these plugins. Example: https://plugins.jenkins.io/singleuseslave/